exercises/medium/secondsToString/secondsToString.js

```diff
@@ -0,0 +1,10 @@
+// Given a number a, representing a number of seconds, return an array of
+// the form [days, hours, minutes, seconds] which is the ammount of time a
+// represents
+// Example:
+// 90061 -> "1 1 1 1" (1 day, 1 hour, 1 min and 1 seconds)
```

Comment:
`1 second` here, no? 

So `0 seconds`, but then `1 second` and all the rest should be plural.

Reply:
Ah, nvm, you're just asking for the numbers output